src/components/AuthComponent/AuthComponent.tsx

```diff
@@ -0,0 +1,12 @@
+import { Navigate, Outlet } from 'react-router-dom';
+
+const useAuth = () => {
+	const user = localStorage.getItem('access_token');
+	return user ? true : false;
```

Comment:
it's not very clear from the name what this hook returns, my first guess would be the authenticated user object

for better clarity you could return an object
`return { isAuthenticated: !!user }`

Reply:
Fixed